Errror in TypeSciprt - Error TS2307: Cannot find module 'angular2-scroll-glue'.

Thanks for the awesome module!

I got this error in an Ionic 2 project based on TypeScript.
`import ScrollGlue from 'angular2-scroll-glue';`
`Error TS2307: Cannot find module 'angular2-scroll-glue'.`

```
$ ionic info

Your system information:

Cordova CLI: 6.1.1
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.6-201604261740
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
ios-deploy version: 1.8.6 
ios-sim version: 5.0.6 
OS: Mac OS X El Capitan
Node Version: v4.3.2
Xcode version: Xcode 7.3.1 Build version 7D1014
```

I googled a bit and found this SO page describing how to write some `.d.ts` file to make it work.
http://stackoverflow.com/questions/22842389/how-to-import-a-js-library-without-definition-file-in-typescript-file

I haven't tried that yet, instead, I'm putting the `examples/app/ScrollGlue.ts` source file into my project to make it work.

It would be great to provide `.d.ts` files (if that's the proper way to do it) and make TypeScript import work.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "angular2-scroll-glue",
   "version": "1.0.6",
   "description": "Angular2 attribute directive for keeping a scrollable div scrolled to a particular position",
-  "main": "webpack.config.js",
+  "main": "index.js",
   "url": "https://github.com/dharness/angular2-scroll-glue",
   "dependencies": {
     "angular2": "2.0.0-beta.9",

